src/workerd/api/node/crypto.c++

```diff
@@ -1129,10 +1126,10 @@ jsg::Optional<CryptoImpl::CipherInfo> CryptoImpl::getCipherInfo(
       }
     }
 
-    auto nameView = cipher.getName();
+    auto nameCstr = cipher.getName();
     auto modeView = cipher.getModeLabel();
-    kj::String name = kj::str(kj::heapArray<char>(nameView.data(), nameView.size()));
-    kj::String mode = kj::str(kj::heapArray<char>(modeView.data(), modeView.size()));
+    kj::String name = kj::heapString(nameCstr);
+    kj::String mode = kj::str(modeView.begin(), modeView.size());
```

Comment:
Having a bit of an end-of-day moment, so I'll just flag this to make sure I recheck if this is right later.